.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,71 @@
+# For most projects, this workflow file will not need changing; you simply need
```

Comment:
If you've moved the CodeQL scan into ci.yml, it should be okay to delete this file :)